strawberryfields/gbs/plot.py

```diff
@@ -0,0 +1,235 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Plotting and visualization
+==========================
+
+**Module name:** :mod:`strawberryfields.gbs.plot`
+
+.. currentmodule:: strawberryfields.gbs.plot
+
+This module provides functionality for visualizing GBS outputs as graphs and point processes.
```

Comment:
Do we want to link it specifically to GBS, e.g., could be just "functionality for visualizing graphs, subgraphs and point processes"?

Reply:
You're right, best not to link specifically to GBS. In fact, would be cool if people use it outside of GBS because it looks so good!